Jakefile.js

```diff
@@ -611,6 +611,7 @@ compileFile(
             fs.readFileSync(tsserverLibraryDefinitionFile).toString() +
             "\r\nexport = ts;" +
             "\r\nexport as namespace ts;";
+        tsserverLibraryDefinitionFileContents = tsserverLibraryDefinitionFileContents.replace(/^(\s*)(export )?const enum (\S+) {(\s*)$/gm, '$1$2enum $3 {$4');

```

Comment:
Can you put the regexp in one location to ensure that any modifications are synchronized?